tests/compile_ouroboros.py

```diff
@@ -0,0 +1,670 @@
+# -*- coding: utf-8 -*-
+"""Check modules from Ouroboros that compile with voc."""
+
+# Standard library imports
+from datetime import datetime
+from subprocess import Popen, PIPE
+import multiprocessing
+import os
+
+HERE = os.path.dirname(os.path.realpath(__file__))
+MODULES_WHITELIST = [
```

Comment:
There will either be a `python/<platform>/python/<module>.java`, or a `python/<platform>/python/<module>/__init__.java`, where `<platform>` is one of `common`, `java` or `android`. 

By chance, one of the modules you've flagged as working already has a native implementation - `types`.